src/match.js

```diff
@@ -19,7 +19,7 @@ export class Match extends Component {
 		return props.children({
 			url,
 			path,
-			matches: path===props.path
+			matches: exec(path, props.path, {}) !== false
```

Comment:
might want to also pass the parameters like a normal route would

Reply:
Yes, that might be a good idea. I do, however, believe it is out of scope for the PR since it would bring about one more change other than the meaning of the `matches` parameter.